prow/config/labels.yaml

```diff
@@ -2773,114 +2871,156 @@ repos:
+      - name: area/migrate
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the migrate SIG.
+        description: Indicates that the Issue or PR belongs to the area of TiDB migration tools.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/bigdata
+      - name: area/ticdc
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the bigdata SIG.
+        description: Indicates that the Issue or PR belongs to the area of TiCDC.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/planner
+      - name: area/br
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the planner SIG.
+        description: Indicates that the Issue or PR belongs to the area of BR (Backup & Restore).
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/sql-infra
+      - name: area/tidb-lightning
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the sql-infra SIG.
+        description: Indicates that the Issue or PR belongs to the area of TiDB Lightning.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/transaction
+      - name: area/dumpling
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the transaction SIG.
+        description: Indicates that the Issue or PR belongs to the area of Dumpling.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/coprocessor
+      - name: area/tidb-binlog
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the coprocessor SIG.
+        description: Indicates that the Issue or PR belongs to the area of tidb-binlog.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/engine
+      - name: area/sync-diff-inspector
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the engine SIG.
+        description: Indicates that the Issue or PR belongs to the area of sync-diff-inspector.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/raft
+      - name: area/bigdata
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the raft SIG.
+        description: Indicates that the Issue or PR belongs to the area of TiFlash, TiSpark, and OLAP.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
```

Comment:
Maybe you can add a `previously` field, which will make the bot help us rename the old label.

```yaml
      - name: area/bigdata
        color: "e99695"
        description: Indicates that the Issue or PR belongs to the area of TiFlash, TiSpark, and OLAP.
        target: both
        prowPlugin: ti-community-label
        isExternalPlugin: true
        addedBy: anyone
        previously:
            - name: sig/bigdata
              color: e99695
```

Ref: https://github.com/kubernetes/test-infra/tree/395804ec9742cc8b5afe8f44830286bcd9821655/label_sync#configuration